adoc/admin-stratos-web-console.adoc

```diff
@@ -0,0 +1,233 @@
+= Stratos Web Console
+
+The Stratos user interface (UI) is a modern web-based management application for
+{kube} and for Cloud Foundry distributions based on {kube} like SUSE
+Cloud Application Platform.
+
+Stratos provides a graphical management console for both developers and system
+administrators.
+
+A single Stratos instance can be used to monitor multiple {kube} clusters
+as long as it is granted access to their {kube} API endpoint.
+
+If you plan to deploy SUSE Cloud Application Platform on your {productname}
+cluster please skip this section of the documentation and refer
+to the official SUSE Cloud Application Platform instructions.
+
+== Introduction
+
+This document aims to describe how to install Stratos in a {productname} cluster
+that doesn't plan to run any SUSE Cloud Application Platform components.
+
+The Stratos stack is deployed using a helm charts and consists of its web
+UI POD and a MariaDB one that is used to store configuration values.
+
+== Prerequisites
+
+=== Persistent Storage
+
+The MariaDB instance used by Stratos requires a persistent storage to store
+its data.
+
+The cluster must have a {kube} Storage Class defined.
+
```

Comment:
Without default storage I had pening status on PVCs.
```
kubectl get pvc -n stratos
NAME                                    STATUS    VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
console-mariadb                         Bound     pvc-cedc6eed-d441-482e-82fd-8c30af174e86   1Gi        RWO            nfs-sc         94m
stratos-console-encryption-key-volume   Pending                                                                                       94m
stratos-console-upgrade-volume          Pending

Events:
  Type    Reason         Age                    From                         Message
  ----    ------         ----                   ----                         -------
  Normal  FailedBinding  4m40s (x362 over 94m)  persistentvolume-controller  no persistent volumes available for this claim and no storage class is set
```
After I made SC to default. all the PVC was bounded.
```
kubectl patch storageclass nfs-sc -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
```
IMO, it is a good idea mentioning making storage class to default.

Reply:
The document already has a section about configuring the storage class to use. I extended this section to mention the usage of a `default` storage class. Thanks